### DISCUSSION TITLE
A question about dates

### DISCUSSION THREAD

**Discussion by DingYiAtGithub8888** [MIT-LCP/mimic-code#1126] [MIMIC-IV]
Dear all,
According the methods of website: “A single date shift was assigned to each subject_id.”
https://physionet.org/content/mimiciv/1.0/

Is it possible to share descriptive statistics of the moving date?
such as maximum, minimum, average, standard deviation, median, 95% confidence interval, etc.

Ignore the year as examples:
before: 2009/03/21
after: 2112/03/25
03/25-03/21=4

before: 2013/09/16
after: 2186/09/13
09/16-09/13=-3

thank you

**Comment by SichengH** [comment]
Not sure what would be the objective of this. Could you please let me know what are you trying to do I might be able to help? Just note that re-identify patients are frowned upon.

**Reply by DingYiAtGithub8888** [reply]
Thanks for your reply.

Our goal is to explore the relationship between disease and time, such as winter flu and summer flu, so the actual date is very important information.
We use different time scales for analysis, e.g. year, seasons(3 months), month, weeks, day, and expect the lowest scale to be weeks but we cannot confirm the lowest ratio.
According to the page, the actual date is within three years. The range is much larger than the season.
https://mimic.mit.edu/docs/iv/modules/core/
"if a patient’s anchor_year is 2158, and their anchor_year_group is 2011 - 2013"

We consider weekly visits as the number of observations.
If the number of moving days is greater than 10 days, then this will has a large observation error.
However, we need to know if such an operation is feasible.

Best,

**Reply by junzaichong666** [reply]
Hi I had the same problem recently and wondering how to get the exact time in the mimic database.  Is your problem solved?

**Reply by DingYiAtGithub8888** [reply]
There is no perfect solution yet.
Before the office releases the offset time range, our team "believes" 1/23 is in winter. On the other hand. Because 1/23 can be in summer or any season. Therefore, the study cannot perform seasonal, monthly, and time series analysis.

**Show again, we 100% agree to the re-identification policy, so no original date is required**
Best,
